### ISSUE TITLE
Not Blocking Ads, DNS Setup Right

### ISSUE THREAD

**Issue by anjesh12** [quidsup/notrack#164]
hello quidsup,
i got a raspberry bi 3 few days ago, and i installed osmc(kodi). On top of that i installed notrack, was successful in installing it. But i am ran into a problem, dns query's are not going though the raspberry pi, and no ads are being blocked,

http://prntscr.com/d2us4w
my setup:
http://prntscr.com/d2uruz
http://prntscr.com/d2us1m

**Comment by quidsup** [comment]
Set the secondary DNS to the same IP 192.168.1.69
Looks like you will have to wait up to 24 hours for the lease to expire on all your devices before they will start using the NoTrack IP.
I don't know why, but running a secondary DNS seems to totally bypass the blocking from NoTrack DNS server

**Comment by anjesh12** [comment]
that fixed the problem, now i am having a second problem. the network is really lagging and sometimes getting ping spikes while playing games

**Comment by mintyfreshpenguin** [comment]
I'm no expert but it looks as though your DNS ip (192.168.1.69) is within the DHCP pool (.64 - .253). Have you set a reservation in the router? Could it be causing an assignment conflict?

**Comment by anjesh12** [comment]
yep, i have the dhcp reservation setup, fixed the problem by switching to google dns, never using opendns. Also i switched over to Raspbian Jessie pixel, for kodi.

**Comment by anjesh12** [comment]
last problem i am having is blocking google ads

http://prntscr.com/d4w27e

**Comment by mintyfreshpenguin** [comment]
You probably need to enable another block list or two - I'm not sure which ones exactly. 

Under Config > Block lists > Advert blocking - I have Easylist and Peter Lowe's Ad server list enabled.

And

Under Config > Black list  - I have enabled doubleclick.net, googletagmanager.com and googletagservices.com. (These were unchecked by default). 


![screenshot at 2016-11-09 09 25 38](https://cloud.githubusercontent.com/assets/21345759/20133503/bae40ca2-a65f-11e6-9550-c649b72550e4.png)

**Comment by anjesh12** [comment]
mintyfreshpenguin
really not sure why we have different results
http://prntscr.com/d5swtu

**Comment by quidsup** [comment]
What happens if you `ping doubleclick.net`

If the IP returned is not your NoTrack device you could force update the blocklists:
On any admin page click the **Options** button in the top-right hand side, then **Update Blocklists**

**Comment by anjesh12** [comment]
```
Microsoft Windows [Version 10.0.10586]
(c) 2015 Microsoft Corporation. All rights reserved.

C:\Users\Admin>ping doubleclick.net

Pinging doubleclick.net [2607:f8b0:400a:806::200e] with 32 bytes of data:
Reply from 2607:f8b0:400a:806::200e: time=21ms
Reply from 2607:f8b0:400a:806::200e: time=21ms
Reply from 2607:f8b0:400a:806::200e: time=21ms
Reply from 2607:f8b0:400a:806::200e: time=21ms

Ping statistics for 2607:f8b0:400a:806::200e:
    Packets: Sent = 4, Received = 4, Lost = 0 (0% loss),
Approximate round trip times in milli-seconds:
    Minimum = 21ms, Maximum = 21ms, Average = 21ms

C:\Users\Admin>
```

**Comment by mintyfreshpenguin** [comment]
That looks like an IPv6 address. Your Notrack device had an IPv4 address didn't it? 
What's the IP of your local system? I'm a bit rusty on Windows, but`ping doubleclick.net -4` should return an IPv4 address.

**Comment by anjesh12** [comment]
```
Microsoft Windows [Version 10.0.10586]
(c) 2015 Microsoft Corporation. All rights reserved.

C:\Users\Admin>ping doubleclick.net -4

Pinging doubleclick.net [209.115.217.25] with 32 bytes of data:
Reply from 209.115.217.25: bytes=32 time=7ms TTL=61
Reply from 209.115.217.25: bytes=32 time=6ms TTL=61
Reply from 209.115.217.25: bytes=32 time=7ms TTL=61
Reply from 209.115.217.25: bytes=32 time=7ms TTL=61

Ping statistics for 209.115.217.25:
    Packets: Sent = 4, Received = 4, Lost = 0 (0% loss),
Approximate round trip times in milli-seconds:
    Minimum = 6ms, Maximum = 7ms, Average = 6ms

C:\Users\Admin>
```

**Comment by mintyfreshpenguin** [comment]
Maybe try clearing the dns cache on your windows machine. then try again.  `ipconfig /flushdns`.
If that fails check the PC is using the correct dns with` ipconfig /all`
